Add plugin metadata documentation and example (#79)
OpenTabletDriver/Plugin-Repository: README.md

| SupportedDriverVersion               | The minimum required OpenTabletDriver version for the plugin to function in four-part version number format.                                                                                                           |
| (Optional) MaxSupportedDriverVersion | The maximum OpenTabletDriver version the plugin functions on in four-part version number format. This should only be included in cases where a minor OpenTabletDriver version introduced breaking changes to a plugin. |
| RepositoryUrl                        | The git repository url containing the plugin's source code.                                                                                                                                                            |
| DownloadUrl                          | The direct download url for the plugin's release files in a compressed archive format. The plugin's dll files must be contained in the top-level directory of the compressed archive.                                  |

> | CompressionFormat                    | The compression format used by the compressed archive in the DownloadUrl field. `zip` is preferred.                                                                                                                    |

Reviewer: ```suggestion
| CompressionFormat                    | The compression format used by the compressed archive in the DownloadUrl field. Only `zip` is supported right now.                                                                                                                    |
```

Reviewer 2: "currently" instead?

Reviewer: Only `zip` is currently supported?

Reviewer 2: Yeah seems better imo

Author: Changed to "Currently, `zip` is the only supported value."

| SHA256                               | The SHA265 hash of the compressed archive in the DownloadUrl field.                                                                                                                                                    |
| (Optional) WikiUrl                   | A link to the plugin's documentation.                                                                                                                                                                                  |

> | LicenseIdentifier                    | The plugin's license name(s).                                                                                                                                                                                          |

Reviewer: ```suggestion
| LicenseIdentifier                    | The plugin's SPDX license identifier(s), separated by comma.                                                                                                                                                                                          |
```

Example plugin metadata:

> ```

Reviewer: ```suggestion
```json
```
